endpoints/delete.js

```diff
@@ -13,7 +13,7 @@ function del(service, urlRoot, router, logger, middleware, emit) {
         }
       } else {
         emit('delete', req)
-        res.send(204)
+        res.sendStatus(204)
```

Comment:
In Express 4 this equivalent method exists (which also sends the plain text string as the response content) http://expressjs.com/en/4x/api.html#res.sendStatus